lms/static/scripts/frontend_apps/api-types.js

```diff
@@ -13,7 +13,7 @@
  * @typedef File
  * @prop {string} id - Identifier for the resource within the LMS's file storage
  * @prop {string} display_name - Name of the resource to present in the file picker
- * @prop {string} updated_at - An ISO 8601 date string
+ * @prop {string} [updated_at] - An ISO 8601 date string
```

Comment:
This was already being treated as optional, and it was only explicitly highlighted by TS once `LMSFilePicker` was migrated.

Reply:
Does this definition agree with the backend code? ie. Are there cases where `updated_at` is not set by the backend, or was the frontend code that treats this as optional incorrect?

Reply:
It's optional in the BE as well, because there was one LMS where this is not available (I would need to check which one)

Reply:
Ok, my mistake. Looks like it's mandatory in the BE, so we may need to check this